src/components/auth-provider.tsx

```diff
@@ -34,26 +33,34 @@ export const AuthProvider: FC = () => {
     const [, getUser] = useAuthAxios<User>({ method: 'GET', url: '/me' }, { manual: true, autoCatch: false })
     const [, requestLogout] = useAuthAxios({ method: 'DELETE', url: '/users' }, { manual: true })
 
-    // Redirect if already logged
-    useEffect(() => {
-        async function checkLoggedIn() {
-            try {
-                const userResponse = await getUser()
-
-                if (currentUser.id !== userResponse.data.id) {
-                  setCurrentUser(userResponse.data)
+    const [checkingUser, setCheckingUser] = useState(true)
+    useEffect(
+        () => {
+            async function checkLoggedIn() {
```

Comment:
If that's an issue? This should be covered by a linting rule. If you insist I can change it though. In the platform team we consider this personal code style, which - when in doubt - should be covered by a linting rule, otherwise it's up to the developer.

Reply:
I very much agree and yes, it should be covered by a linting rule